internal/ignition/ignition.go

```diff
@@ -348,7 +346,7 @@ const discoveryIgnitionConfigFormat = `{
 
 const secondDayWorkerIgnitionFormat = `{
 	"ignition": {
-	  "version": "3.1.0",
+	  "version": "3.2.0",
```

Comment:
Will it work with 3.1.0 as well?
If not we are breaking API here.

Reply:
I did manually test it on a spoke with OCP 4.8; as in the other comment - providing an override with 3.1.0 does work because I'm not requiring a strict match, but allow for backwards-compatible ignition override